### DISCUSSION TITLE
WAVE_CORRECT_AUTO only works in more recent OpenCV versions

### DISCUSSION THREAD

**Discussion by JimReno** [OpenStitching/stitching#101] [General]
My previous opencv-python version is 4.2.0.32. When I run the code i come to an error:
`cv2.detail' has no attribute 'WAVE_CORRECT_AUTO`.

I've upgrade the opencv-python version to 4.7.0.72 to solve the problem. I opened a discussion here in case anyone may come to this error.
The `requirements.txt` file might be update:)

**Comment by lukasalexanderweber** [comment]
Thanks for letting us know!
